pkg/apply/apply.go

```diff
@@ -74,10 +74,50 @@ func ApplyObject(ctx context.Context, client k8sclient.Client, obj *uns.Unstruct
 			}
 
 			return errors.Wrapf(err, "could not update object %s", objDesc)
-		} else {
-			log.Print("update was successful")
 		}
 	}
 
+	log.Print("update was successful")
```

Comment:
If the condition from 56 if false, there is no need to update and we don't need to log. Please keep it on the original place.

Reply:
you right, please re-review that snippet, wrote there a comment (this one is outdated)